### DISCUSSION TITLE
How to create architecture-independent package?

### DISCUSSION THREAD

**Discussion by steve8x8** [goreleaser/goreleaser#4423] [Q&A] [answered]
In some situations, a full-fledged (and architecture-dependent) binary package needs to be accompanied by a helper package that only provides configuration files and/or symbolic links. Neither of those are architecture-dependent, so they could be packaged into a arch-indep package instead. For Debian, that would be `..._all.deb` instead of the usual `..._{{ .Arch }}.deb`. 
I couldn't find any documentation on how to do this, and the projects using GoReleaser I looked into would produce one such complementary package per architecture - which is both confusing and unnecessary. Are there some (hide to me) pointers, or is this a feature not yet included?

**Comment by caarlos0** [comment, marked as answer]
I believe you setting `nfpm.meta=true` does what you need

https://goreleaser.com/customization/nfpm/?h=meta
